app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -26,12 +26,13 @@ def update
 
       form.update!(form_data: params[:form_data] || params[:formData], metadata: params[:metadata])
 
+      veteran_icn = @current_user.icn
       if Flipper.enabled?(:intent_to_file_lighthouse_enabled, @current_user) && form.id_previously_changed? &&
          Lighthouse::CreateIntentToFileJob::ITF_FORMS.include?(form.form_id)
-        if @current_user.participant_id.blank?
+        if veteran_icn.present? && @current_user.participant_id.blank?
```

Comment:
Is this right?  Should it be !veteran_icn.present?

Reply:
my understanding of this issue was that if ICN is missing, then PIDs will always be blanks, so no need to log those instances.  We should only be logging instances where ICN is present,  but PID is missing. Let me know if I've misunderstood something.

Reply:
OK, could be written like this?
`if !veteran_icn.present? || @current_user.participant_id.blank?`